src/components/data-management/ProjectDetails.jsx

```diff
@@ -222,10 +227,13 @@ const ProjectDetails = ({ width, height }) => {
   ];
 
   useEffect(() => {
-    if (samples.ids.length === 0 || projects.ids.length === 0) return;
+    if (samples.ids.length === 0 || projects.ids.length === 0) {
+      setData([]);
+      return;
+    }
 
     const statuses = ['uploaded', 'uploading', 'uploadError', 'fileNotFound'];
-    const newData = projects[activeProjectUuid].samples.map((sampleUuid, idx) => ({
+    const newData = projects[activeProjectUuid]?.samples.map((sampleUuid, idx) => ({
       key: idx,
       name: samples[sampleUuid].name,
```

Comment:
Maybe add an optional chaining for the access to `name`, following the logic of `projects[activeProjectUuid]?.samples`.